server/configs/application.properties

```diff
@@ -44,6 +48,26 @@ mail.smtpUser=@@smtpUser@@
 # mail.smtpSocketFactoryClass=@@smtpSocketFactoryClass@@
 # mail.smtpAuth=@@smtpAuth@@
 
+# Optional - JMS configuration for remote ActiveMQ message management for distributed pipeline jobs
+# https://www.labkey.org/Documentation/wiki-page.view?name=jmsQueue
```

Comment:
Have we captured in a TODO list somewhere that we'll need to update these docs for embedded?

Reply:
Yes, there's docs handoff task in the story on the board. I'll add pointers to these two pages as definitely needing attention in the spec.